nautobot/docs/user-guides/git-data-source.md

```diff
@@ -79,6 +78,10 @@ Fill out the details for the Git repository. More information on the inputs can
 
 As soon as you click on **Create**, Nautobot will clone and sync the repository and provide status of the job.
 
+!!! note
```

Comment:
This isn't ideal as a note in a *user* guide, IMHO - the user may not have the ability to set environment variables for Nautobot. I'm not sure how better to phrase it though, perhaps something like "you will need make sure that the Nautobot installation has set the environment variable..."?

Reply:
`If you are using a self-signed git repository, please refer to the <insert link> development guide.` ?